Yank/experiment.py

```diff
@@ -1359,12 +1390,14 @@ def _system_files(files):
 
         # System schema.
         system_schema = Schema(Or(
-            {'receptor': is_known_molecule, 'ligand': is_known_molecule,
-             'solvent': is_pipeline_solvent, Optional('pack', default=False): bool,
-             Optional('leap'): self._LEAP_PARAMETERS_SCHEMA},
+            And({'receptor': is_known_molecule, 'ligand': is_known_molecule,
+                 'solvent': is_pipeline_solvent, Optional('pack', default=False): bool,
+                 Optional('leap'): self._LEAP_PARAMETERS_SCHEMA},
+                lambda d: check_regions_clash(d, 'ligand', 'receptor')),
 
-            {'solute': is_known_molecule, 'solvent1': is_pipeline_solvent,
-             'solvent2': is_pipeline_solvent, Optional('leap'): self._LEAP_PARAMETERS_SCHEMA},
+            And({'solute': is_known_molecule, 'solvent1': is_pipeline_solvent,
+                'solvent2': is_pipeline_solvent, Optional('leap'): self._LEAP_PARAMETERS_SCHEMA},
+                lambda d: check_regions_clash(d, 'solute')),
```

Comment:
I'm not sure I understand. If we have only one molecule, is it possible to have region name clashes at all?

Reply:
No, but I imagine it would not be hard to extend this formalism to allow regions defined in the *system* as well, if so, we will want to keep this framework here. Its a pretty cheap action but I could remove it if you think it would be best

Reply:
Ah, I see. Yeah, just leave it there then.